src/pages/checboxPrueba.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <p>  {{ string2 }} </p>
+  <Checkbox v-model="isSelected">
+    {{ string + isSelected }}
```

Comment:
(Esto no es algo que tengas que cambiar, pero para que sepas que se puede hacer así también, por si no lo sabías):

```suggestion
  <p>I am checked:</p>
  <Checkbox v-model="isSelected">
    Using the custom checkbox as stand alone {{ isSelected }}
```

Reply:
Eso lo puse así, porque al poner texto plano daba error la comprobación del lint, por lo que lo dejare así.

Reply:
Ah cierto. Igualmente esta página se eliminará una vez terminemos de construir las vistas, porque al fin y al cabo es de prueba.

Por si no te lo has visto todavía, en Discord he subido un vídeo explicando como corregir dicho error y traducir la aplicación correctamente (pronto añadiré los enlaces de todos los vídeos que hago al readme).